docs/UserGuide.md

```diff
@@ -320,6 +317,28 @@ Examples:
 * `list` followed by `share 2` copies the information of the second contact in the list displayed by SociaLite.
 * Clicking on the "Share" button of Annie Baker's name card copies her contact information.
 
+<br>
+
+### Pinning and unpinning a contact card: `pin`/`unpin`
+
+Pins/Unpins the contact card of the specified person to the system clipboard. The pinned contact cards will be alphabetically ordered at the top of the person list.
+
+Alternatively, clicking on the "Pin/Unpin" button at the top-right hand corner of the specific contact card will also pin the contact card.
+
+Format: `pin INDEX`/`unpin INDEX`
+
+* Pins the contact card of the person at the specified `INDEX` at the top of the person list
+* The index refers to the index number shown in the displayed person list
+* The index **must be a positive integer** 1, 2, 3, ...
+* To pin/unpin a contact card, the contact card must be unpinned/pinned.
+
+Examples:
+
+* Provided no contact card has been pinned yet, `list` followed by `pin 3` pins the third contact in the list displayed by SociaLite and places it at the top of the person list.
+* Clicking on the "Pin/Unpin" button of Annie Baker's name card unpins her contact card if has been previously pinned.
```

Comment:
Can consider modifying to: "... unpins her contact card if it was previously pinned."